.readthedocs.yml

```diff
@@ -0,0 +1,23 @@
+# .readthedocs.yml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# RTD configuration file version
+version: 2
+
+# Build documentation in the docs/ directory with Sphinx
+sphinx:
+  builder: html
+  configuration: docs/source/conf.py
+  fail_on_warning: true
+
+# Optionally build your docs in additional formats such as PDF
+formats: []
+
+# Optionally set the version of Python and requirements required to build your docs
+python:
+  version: 3.6
+  install:
+    - method: pip
+      path: .
+    - requirements: docs/requirements.txt
```

Comment:
This means we have some abiguity regarding "building the docs" now, right? What I mean is that tox does not perfectly mirror the way the docs are built anymore.

Reply:
I wasnt able to figure out a way to have ReadTheDocs use tox. ReadTheDocs also doesnt seem to support using a custom Makefile to build the docs (see also: https://github.com/readthedocs/readthedocs.org/issues/2423). I think the closest we can get for reproducibility is to specify a common `docs/requirements.txt`. ReadTheDocs then calls `sphinx-build` like the makefile, but potentially with different options which might cause divergent behaviour, I guess.

Reply:
I think it should be fine this way :) Maybe one thing we could do is to have tox use the same `./docs/requirements.txt` file

Reply:
Okay I just realized that you already adjusted `tox.ini`, awesome!